packages/frontend/src/app/repository/tests-list/test-details/test-details.component.ts

```diff
@@ -14,17 +14,37 @@
 
 import {Component, Input} from '@angular/core';
 import {Test} from 'src/app/services/search/interfaces';
+import {COMService} from '../../../services/com/com.service';
+import {Observable} from 'rxjs';
 
 @Component({
   selector: 'app-test-details',
   templateUrl: './test-details.component.html',
   styleUrls: ['./test-details.component.css'],
 })
 export class TestDetailsComponent {
+  constructor(public comService: COMService) {}
   @Input() test: Test;
+  @Input() repoName: string;
+  @Input() orgName: string;
+  windowProvider = window;
+  public deleteUrl$: Observable<string>;
 
   //Converts tests' passing percentage from decimal to percentage
   toPercentage(percentpassing: number): string {
     return (percentpassing * 100).toFixed(2);
   }
+
+  startDeleteTest() {
+    this.deleteUrl$ = this.comService.fetchDeleteTestUrl(
+      this.orgName,
+      this.repoName,
+      this.test.name,
+      'https://flaky-dashboard.web.app/'
```

Comment:
Should this link come from `environment.baseUrl`?
Otherwise will it always redirect to the `production` website?